### PR TITLE
Improvement/zenko 3835/lifecycle tests

### PR DESCRIPTION
**Pull request by alexanderchan-scality**
**What does this PR do, and why do we need it?**

**Which issue does this PR fix?**

fixes #<ISSUE>

**Special notes for your reviewers**:

### CODE REVIEW

**Comment by XinLi0207** [comment]
hi Alex, I found that in the backbeat v8.3.6 and latest dev/8.3 branch, the cloudserver is 8.3.10, not 8.4.x, it was reverted in this integration commit
[https://github.com/scality/backbeat/commit/70d1d493065e931926f20f5ae6b9bb19eca1ef36#diff-7ae45ad102eab3b6d7e7896ac[…]5b346470d7bc6507b6481575d519L66](https://github.com/scality/backbeat/commit/70d1d493065e931926f20f5ae6b9bb19eca1ef36#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L66)
I just found here weird, I don't know if it's related or not, should we update cloudserver version to 8.4.x to keep them both the latest?

**Comment by alexanderchan-scality** [comment]
hi @XinLiScality
can't say much about the dev dependency revert in backabeat as I don't have much context for it, but that should not affect the dep versions pinned in zenko.

The dependency bump in this PR however is to have the minimum dep changes to deploy a functioning lifecycle feature, which only includes bumping backbeat to `8.3.8` for lifecycle related changes.

**Review by nicolas2bert — APPROVED** [review]

File: tests/zenko_tests/node_tests/package.json [hunk 1]
```diff
@@ -51,7 +52,7 @@
     "test_operator": "mocha --tags ${MOCHA_TAGS} --exit -t 10000 --reporter mocha-multi-reporters --reporter-options configFile=config.json ./init_test.js",
     "test_smoke": "mocha --tags ${MOCHA_TAGS} --exit -t 10000 --reporter mocha-multi-reporters --reporter-options configFile=config.json --recursive smoke_tests",
     "test_iam_policies": "mocha --tags ${MOCHA_TAGS} --exit -t 10000 --reporter mocha-multi-reporters --reporter-options configFile=config.json --recursive iam_policies",
-    "test_all_extensions": "npm-run-all test_aws_crr test_ingestion_oob_s3c",
```
Comment (rachedbenmustapha) [inline tests/zenko_tests/node_tests/package.json:54]: Is this removal expected?

**Review by anurag4DSB — APPROVED** [review]

**Review by rachedbenmustapha — APPROVED** [review]

**Review by rachedbenmustapha — APPROVED** [review]

File: tests/zenko_tests/node_tests/package.json (hunk 1 above)
Comment (alexanderchan-scality) [reply tests/zenko_tests/node_tests/package.json:54]: at the time of removing this, there was an issue with the ring environment. as to not block this PR. i had created a commit to temporarily remove the ingestion test.

**Pull request merged**